### PR TITLE
Allow augmentations to accept a list of files

### PR DESCRIPTION
**Pull request by atamazian**
Attempt to implement https://github.com/iver56/audiomentations/issues/187 , with application to other augmentations as well. Replaced `get_file_paths()` with `find_audio_files_in_paths()` from `torch-audiomentations`.

### CODE REVIEW

File: audiomentations/augmentations/trim.py [hunk 1]
```diff
@@ -19,22 +19,10 @@ def __init__(self, top_db: Optional[float] = None, p: Optional[float] = None):
...
 
         if p is None:
-            p = 1.0
-            warnings.warn(
-                "The default value of p in Trim will change from 1.0 to 0.5 in a future"
-                " version of audiomentations. Please specify it explicitly to make your"
-                " code more future-proof and to get rid of this warning.",
-                FutureWarning
-            )
+            p = 0.5
```
Comment (iver56) [inline audiomentations/augmentations/trim.py:25]: Not ready for these changes yet. Please revert this

File: audiomentations/core/utils.py [hunk 2]
```diff
@@ -16,31 +16,36 @@
...
 )
-
-
-def get_file_paths(
-    root_path: Union[str, Path],
-    filename_endings=AUDIO_FILENAME_ENDINGS,
+    
+def find_audio_files_in_paths(
+    paths: Union[List[Path], List[str], Path, str],
+    filename_endings=SUPPORTED_EXTENSIONS,
```
Comment (iver56) [inline audiomentations/core/utils.py:22]: `E   NameError: name 'SUPPORTED_EXTENSIONS' is not defined`

**Review by iver56 — CHANGES_REQUESTED** [review]

File: audiomentations/core/utils.py (hunk 2 above)
Comment (atamazian) [reply audiomentations/core/utils.py:22]: Fixed.

**Comment by iver56** [comment]
I ran the tests on this branch and I see many test failures like this:

```
>               file_paths += find_audio_files(
                    p,
                    filename_endings=filename_endings,
                    traverse_subdirectories=traverse_subdirectories,
                    follow_symlinks=follow_symlinks
                )
E               NameError: name 'find_audio_files' is not defined
```

**Comment by atamazian** [comment]
I've fixed this. Can you check again?

**Comment by iver56** [comment]
That fix looks incorrect, as you're trying to adopt the approach in https://github.com/asteroid-team/torch-audiomentations/blob/master/torch_audiomentations/utils/file.py#L43 and not do recursion

**Comment by atamazian** [comment]
Is there any easy way to run all tests at once without importing each testing class?

**Comment by iver56** [comment]
Yes, you just run `pytest`

https://github.com/iver56/audiomentations#run-tests-and-measure-code-coverage

**Comment by atamazian** [comment]
Many thanks!

**Review by iver56 — APPROVED** [review]
Thanks, LGTM now

**Pull request merged**